back/libs/oidc-client/src/exceptions/oidc-client-missing-code.exception.ts

```diff
@@ -1,15 +0,0 @@
-import { HttpStatus } from '@nestjs/common';
-
-import { ErrorCode } from '../enums';
-import { OidcClientBaseException } from './oidc-client-base.exception';
-
-export class OidcClientMissingCodeException extends OidcClientBaseException {
-  public code = ErrorCode.MISSING_CODE;
```

Comment:
nitpick: we could also remove this enum (and any non used enum) in back/libs/oidc-client/src/enums